lib/simplifyExpression/fractionsSearch/divideByGCD.js

```diff
@@ -50,6 +50,7 @@ function divideByGCD(fraction) {
     '/', [intermediateNumerator, intermediateDenominator]);
   const status = Node.Status.nodeChanged(
     ChangeTypes.FIND_GCD, fraction, intermediateFraction);
+  status.newNode = Node.Status.resetChangeGroups(status.newNode);
```

Comment:
you can set it manually I think

change groups are usually set by default here
https://github.com/socraticorg/mathsteps/blob/master/lib/node/Status.js#L50
which you turn off by passing in false

and then you can just set changeGroup=1 on both the "3" nodes

...does that make sense? It's kind of weird thing so lemme know if you want more explanation